core/src/main/scala/kafka/server/ReplicaFetcherBlockingSend.scala

```diff
@@ -49,6 +49,7 @@ class ReplicaFetcherBlockingSend(sourceBroker: BrokerEndPoint,
   private val socketTimeout: Int = brokerConfig.replicaSocketTimeoutMs
 
   private val networkClient = {
+    val logContext = new LogContext("[ReplicaFetcherBlockingSend clientId=" + clientId + "] ")
```

Comment:
I would pass this from `ReplicaFetcherThread`. We can take the chance and set a reasonable `logIdent` there too.